test/dump/extended-names.txt

```diff
@@ -204,10 +208,12 @@ Custom:
  - memory[0] <mem2>
  - global[0] <g1>
  - global[1] <g2>
+ - elemseg[0] <elem1>
```

Comment:
Where does this "elem1" come from?

Reply:
The out of order name section meant is wasn't being displayed here.